plugins/tabletools/dialogs/tableCell.js

```diff
@@ -518,6 +412,76 @@ CKEDITOR.dialog.add( 'cellProperties', function( editor ) {
 		};
 	}
 
+	function getCellSizeFieldDefinition( fieldName ) {
+		return {
+			requiredContent: 'td{' + fieldName + '}',
+			type: 'hbox',
+			widths: [ '70%', '30%' ],
+			children: [ {
+				type: 'text',
+				id: fieldName,
+				width: '100px',
+				label: langCommon[ fieldName ],
+				validate: validate.number( langCell[ 'invalid' + CKEDITOR.tools.capitalize( fieldName ) ] ),
+
+				// Extra labelling of unit type.
+				onLoad: function() {
+					var unitType = this.getDialog().getContentElement( 'info', fieldName + 'Type' ),
+						labelElement = unitType.getElement(),
+						inputElement = this.getInputElement(),
+						ariaLabelledByAttr = inputElement.getAttribute( 'aria-labelledby' );
+
+					inputElement.setAttribute( 'aria-labelledby', [ ariaLabelledByAttr, labelElement.$.id ].join( ' ' ) );
+				},
+
+				setup: setupCells( function( element ) {
+					var attr = parseFloat( element.getAttribute( fieldName ), 10 ),
+						style = parseFloat( element.getStyle( fieldName ), 10 );
+
+					if ( !isNaN( style ) ) {
+						return style;
+					} else if ( !isNaN( attr ) ) {
```

Comment:
When there is a return statement inside `if`, `else` keyword is redundant.

```suggestion
					}
					if ( !isNaN( attr ) ) {
```